### PR TITLE
Adjust spacing a bit to reduce whitespace in Settings

### PR DESCRIPTION
**Pull request by rickpasetto**
This makes the settings view (at least to my eyes) more like the design spec.

### CODE REVIEW

File: Loop/Views/SettingsView.swift [hunk 1]
```diff
@@ -167,9 +168,9 @@ fileprivate struct LargeButton: View {
 
     // TODO: The design doesn't show this, but do we need to consider different values here for different size classes?
     static let spacing: CGFloat = 15
-    static let imageWidth: CGFloat = 48
-    static let imageHeight: CGFloat = 48
-    static let topBottomPadding: CGFloat = 20
+    static let imageWidth: CGFloat = 60
+    static let imageHeight: CGFloat = 60
```
Comment (nhamming) [inline Loop/Views/SettingsView.swift:172]: [nit] should accessibility be considered for this image? It is already pretty large, but thought I should ask.

**Review by nhamming — APPROVED** [review]
LGTM

**Review by novalegra — APPROVED** [review]
👍

File: Loop/Views/SettingsView.swift (hunk 1 above)
Comment (rickpasetto) [reply Loop/Views/SettingsView.swift:172]: I'm going to assume it is large enough... and count on design review to catch that if need be.

**Pull request merged**